msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -11,22 +11,33 @@ ms.localizationpriority: high
 
 > [!NOTE]
 >
-> * Bot configuration experience is available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
-> * Bot configuration experience is supported in channel or group chat scopes only.
+> Bot configuration experience is supported in channel or group chat scopes only.
 
-You can create a bot to enable the bot configuration settings for the user during the bot installation and also from the channel or group chat scope after the bot is installed.
+When you're creating a bot for Teams, it’s important to make sure users have a smooth experience and provision control to personalize their bots in places like Microsoft Teams. The ability to configure bots post-installation within Teams allows developers to fine-tune bot functionality according to user requirements even after deployment. You can use methods from the Microsoft Bot Framework SDK to implement bot configuration features that enhance user engagement and productivity. 
+ 
+In this article, we'll explore methods for configuring bots post-installation, both for cross-platform compatibility and specific to Teams. You can create complex bot configurations that enhance user engagement and efficiency.
 
-In the following graphic, the bot is installed in a group chat. When the user hovers over the bot, an Adaptive Card appears. The user can select the settings icon in the Adaptive Card to update or change the bot's configuration settings:
+There are two ways to initiate bot configuration settings post-installation:
 
-:::image type="content" source="../../assets/images/bots/configurationbot.gif" alt-text="Screenshot shows the configuration option for the bot in a Teams group chat.":::
+1. @mention the bot in the message compose area.
+
+   :::image type="content" source="../../assets/images/bots/mention-bot-in-compose-area.gif" alt-text="Screenshot shows the configuration option for the bot in the message compose area.":::
+
+1. When the user hovers over the bot, an Adaptive Card appears. The user can select the settings icon in the Adaptive Card to update or change the bot's configuration settings.
```

Comment:
Does the settings icon appear for *all* bots, or only those that have opted in to the configuration mechanism by adding the node to their manifest? The sentence as worded implies that all bots get this, without qualification.